foundations/tying_it_all_together/conclusion.md

```diff
@@ -1,43 +1,44 @@
 That was quite a journey.  Take a second and reflect on the fact that you've just learned the basics of at least three different languages and, if you had to, you could probably hack together a web page on your own already.  
 
-You should be fairly comfortable typing commands into your computer's command line and you should understand what happens when you enter a web address into your browser and it returns a functioning page.  You should be able to create a basic web page and make it do some basic things with JavaScript.  You should understand the purpose of backend code and how frameworks make it easier to write clean and maintainable code.  Finally, you should have a clear understanding of how all this fits together in your future as a builder of web applications.
+You should be fairly comfortable typing commands into your computer's command line and you should understand what happens when you enter a web address into your browser and it returns a page.  You should be able to create a basic web page and make it interactive with JavaScript.  You should understand the purpose of backend code and how frameworks make it easier to bootstrap new projects.  Finally, you should have a clear understanding of how all this fits together in your future as a builder of web applications.
```

Comment:
instead of "interactive with javascript" it should say "interactive using javascript"

A beginner has no idea what "bootstrap new projects" means. I think this should be left as-is.

Reply:
"Bootstrap" is not technical jargon. And I disagree that frameworks inherently make your code cleaner or more maintainable. You're right about "interactive *with*" vs "interactive *using*" though.